swan-lake/development-tutorials/source-code-dependencies/customizing-formatting.md

```diff
@@ -0,0 +1,389 @@
+---
+title: Customization of formatting
+description: The section on providing custom configurations to the formatter.
+keywords: ballerina, programming language, formatting, custom configurations
+permalink: /learn/customizing-formatting/
+active: customizing-formatting
+intro: This feature allows developers to customize the formatting of Ballerina projects. This is done with the use of a configuration file. This configuration file can be a local or a remote file, which allows for consistency in code style across projects and simplifies the process of enforcing formatting standards. This is introduced as an experimental feature in Ballerina 2201.9.0.
+---
+
+## Configuring
+
+Users can provide custom formatting configurations with the use of a separate configuration file of the `toml` format. Users can provide an arbitrary name for the file.
+
+The formatting configuration file can be specified in two ways:
+
+* `Ballerina.toml`: Define the path to a configuration file within the `Ballerina.toml` itself. The path can be absolute, relative to the package root, or a URL (HTTP/HTTPS). This path is specified under the `format` section in `Ballerina.toml` with the use of the `configPath` attribute.
+
+    ```
+    [format]
+    configPath = "Format.toml"
+    ```
```

Comment:
````suggestion
    ```toml
    [format]
    configPath = "Format.toml"
    ```
````

May also make sense to have a more complete file, rather than just this bit.